content/news/events-january-2025.md

```diff
@@ -0,0 +1,82 @@
++++
+headline = "ARTICLE"
+title = "Foulab Events - January 2024"
++++
+
+{{< centered-image src="events-jan-2025.gif" width="384">}}
+
+
+<center><i>An early 3D model of the workshop by **v2thegreat**</i></center>
+
+<br/>
+
+Do you feel it? We're a quarter of the way through the 21st century! Feel old yet? Surpress those feelings with some good old-fashioned hacking and making! Here's what's coming up in January:
+
+<br/>
+
+
+### Wednesday Mesh Evening - Wednesday January 8th @ 19:00 - 22:00
+
+Tired of spending time with your family? I know I am. Join us for the first social gathering of the year! Lots of new nodes, the region's Meshtastic coverage is booming - what a good time to mesh around this January is shaping up to be!
+
+Join the network! Explore mesh networking in all its forms. Discuss, research, test the unknown, and deploy peer-to-peer infrastructure!
+
+Everyone is hosting, and **kryma** invites you to host as well. 
+
+The network must grow! The network must grow! The network must grow! The network must grow! The network must grow! The network must grow! The network mu- (the author has been disconnected from the network)
+
+<br/>
+
+
+### Thursday, January 9th, 8:00PM to 9:30PM: Workshop Reorganization Planning!
+
+The foulab workshop area needs some love! We have a lot of tools, both traditional (hand tools, drill press, bandsaw, air compressor, welder, etc) and modern (3D printer, CNC mills, laser cutter) but not a lot of good places to put them.
```

Comment:
I suggest stylize as 'Foulab' (capital F), consistent with the style in most places.